composer.json

```diff
@@ -29,11 +29,11 @@
         "php": ">=5.4",
         "doctrine/common": "~2.4",
         "doctrine/orm": "~2.4",
-        "symfony/http-kernel": "~2.4",
```

Comment:
Can you change this to `^2.4|^3.0`?
Here is the same, but I feel more comfortable working with `^` :)